releases/release-5.1.2.md

```diff
@@ -0,0 +1,156 @@
+---
+title: TiDB 5.1.2 Release Notes
+---
+
+# TiDB 5.1.2 Release Notes
+
+Release Date: September 27, 2021
+
+TiDB version: 5.1.2
+
+## Improvements
+
++ TiDB
+
+    - Trigger auto-analyze based on histogram row count [#26708](https://github.com/pingcap/tidb/pull/26708)
+
++ TiFlash
+
+    - Support pushing down ROUND(X) to TiFlash. [#2611](https://github.com/pingcap/tics/pull/2611)
+    - support function DATE() pushed down to tiflash [#2602](https://github.com/pingcap/tics/pull/2602)
+    - support functions of ADDDATE() and DATE_ADD() pushed down to tiflash [#2591](https://github.com/pingcap/tics/pull/2591)
+    - retry when meeting stablish conn fails [#2570](https://github.com/pingcap/tics/pull/2570)
+    - Support pushing down MOD(N, M) to TiFlash. [#2560](https://github.com/pingcap/tics/pull/2560)
+    - Add Grafana panels for write throughput per instance [#2544](https://github.com/pingcap/tics/pull/2544)
+    -  Cherry-pick #2216 #2460 #2494 & Optimizr read-index retry to release-5.1 [#2345](https://github.com/pingcap/tics/pull/2345)
+
```

Comment:
+ PD
- dynamically adjust the retry limit according to the operator [#4048](https://github.com/tikv/pd/pull/4048)